_posts/2018-03-15-release-3-3-0.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Release 3.3.0"
+date: 2018-03-16 10:00:00
+author: "@mikaelrandy"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.3.0][milestone]!
+
+## Stats
+
+* 43 [commits][commits],
+* 129 files changed,
+* 7 contributors,
```

Comment:
I got 6 contributors in my side ?

I think adding also the fact that we have also theses stats
* 7 [new features][changelog],
* 5 [bug fix][changelog].